src/backends/utils.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
+
+const _ = require('lodash')
+const config = require('../services/config')
+
+function getCurrencyPair (source_ledger, destination_ledger) {
+  if (!_.isEmpty(config.get('tradingPairs'))) {
+    for (let pair of config.get('tradingPairs')) {
+      // trading pair is of the form
+      // ["<currency@<source_ledger>","<currency>@<destination_ledger>"]
+      if (pair[0].indexOf(source_ledger) === 4 &&
+        pair[1].indexOf(destination_ledger) === 4) {
+        return [pair[0].slice(0, 3), pair[1].slice(0, 3)]
+      }
+    }
+  }
+  // No currency pair found
+  return ['', '']
+}
+
+/**
+ * Check if we trade the given pair of assets
+ *
+ * @param {Array.<Array.<String>>} Array of pairs
+ *   of currencies traded by this connector
+ * @param {String} source The URI of the source ledger
+ * @param {String} destination The URI of the destination ledger
+ * @return {boolean}
+ */
+function hasPair (currencyPairs, source, destination) {
```

Comment:
Actually @clark800 nice catch ... it looks like the two backends structured the pairs differently. It was a flat array in fixer.io and an array of arrays (each array being a pair) in ilp-quoter. I changed this to allow the more general case of trading more than one currency pair. All the more reason to have unit tests for the ilp-quoter backend. There is some larger cleanup that could be done with the backends, but that's out of the scope of this PR.